packages/Picker/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { forwardRef } from 'react'
+import { array, bool, func, string } from 'prop-types'
+import { Box } from '@xstyled/styled-components'
+import { RadioGroup as ReakitRadioGroup, useRadioState } from 'reakit/Radio'
+import { FieldGroup } from '@welcome-ui/field-group'
+import { Label } from '@welcome-ui/label'
+
+import * as S from './styles'
+
+export const Picker = forwardRef(
+  ({ connected, label, name, onChange, options, required, value }, ref) => {
+    const radio = useRadioState({ state: value })
+
+    const handleClick = e => {
+      e.stopPropagation()
+      radio.onClick && radio.onClick(e)
+      onChange && onChange(e)
+    }
+
+    return (
+      <FieldGroup as={ReakitRadioGroup} mb={0} ref={ref} required={required}>
```

Comment:
add the possibility to add styled system props on this components
```suggestion
      <FieldGroup as={ReakitRadioGroup} mb={0} ref={ref} required={required} {...rest}>
```